examples/playground/main.rs

```diff
@@ -42,6 +43,25 @@ impl Scene for MainScene {
             }),
         );
     }
+
+    fn handle_event(&mut self, _context: Context, event: Event<()>) -> Transition<()> {
+        match event {
+            Event::NewEvents(_) => {}
+            Event::WindowEvent { window_id, event } => {
+                println!("WINDOW: {:?} -> EVENT: {:?}", window_id, event)
+            }
+            Event::DeviceEvent { .. } => {}
+            Event::UserEvent(_) => {}
+            Event::Suspended => {}
+            Event::Resumed => {}
+            Event::MainEventsCleared => {}
+            Event::RedrawRequested(_) => {}
+            Event::RedrawEventsCleared => {}
+            Event::LoopDestroyed => {}
+        }
```

Comment:
Why do we need to specify all the specialisations (if this is how they are called in Rust) if we are only handling `WindowEvent`? Can we not do a broader pattern that fits everything that is not a `WindowEvent`? (In OCaml you would do a `_` -> `blah`)

Reply:
No we do not need to match all enum variants here.

I literally matched everything because I autocompleted the match arms in my IDE.

I do not think that all those events should propagate to scenes. Maybe defining our own Event enum and wrapping/converting winit's enum in order to have a cleaner API. I will think of a way to remove not needed event variants as I inceremntally improve this draft PR.

And yes we can use `_ => {}` for all cases we do not care about.